### PR TITLE
Remove kebabCase on generateSeparateFile

### PR DESCRIPTION
**Pull request by fmvilas**
It removes `_.kebabCase` from the name of the file on the `generateSeparateFile` function. It was conflicting with file names like `MyClass.java`.

### CODE REVIEW

**Comment by fmvilas** [comment]
You're right. We should probably convert names into filename-safe strings.

File: package.json [hunk 1]
```diff
@@ -31,13 +31,12 @@
...
   "dependencies": {
     "ajv": "^6.10.2",
-    "asyncapi": "^2.2.0",
     "asyncapi-parser": "^0.12.0",
     "commander": "^2.12.2",
+    "filenamify": "^4.1.0",
     "fs.extra": "^1.3.2",
     "jmespath": "^0.15.0",
     "js-yaml": "^3.13.1",
-    "json-schema-ref-parser": "^5.1.0",
```
Comment (fmvilas) [inline package.json]: Took the opportunity to remove unused dependencies.

**Comment by fmvilas** [comment]
Not really, only when invalid chars are found. For instance, if your schema name is `MySchema` it will remain as `MySchema`. In the case of a channel like `my/channel` then yes, it would become `my-channel`.

**Comment by jonaslagoni** [comment]
> Not really, only when invalid chars are found. For instance, if your schema name is `MySchema` it will remain as `MySchema`. In the case of a channel like `my/channel` then yes, it would become `my-channel`.

But since that problem is that `my-channel` would not work in java, shouldn't this be converted to `MyChannel` :smile: ?

**Comment by fmvilas** [comment]
The problem is on schemas and other objects like messages. I don't think channels will need to have their own class file. If they do, we can add an option later.

**Pull request merged**